app/Ticket.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace App;
```

Comment:
```suggestion
namespace App/Models;
```
Chuyển hết Model vào thư mục Models